enterprise/views.py

```diff
@@ -1404,21 +1404,24 @@ def get_course_run_id(enterprise_customer, course_key):
         else:
             raise Http404
 
-    def eligible_for_direct_audit_enrollment(self, request, enterprise_customer, course_run_id):
+    def eligible_for_direct_audit_enrollment(self, request, enterprise_customer, resource_id, course_key=None):
         """
         Return whether a request is eligible for direct audit enrollment for a particular enterprise customer.
 
+        'resource_id' can be either course_run_id or program_uuid.
```

Comment:
nit: use backquotes while quoting variables in docstrings 
```"`resource_id` can be either `course_run_id` or `program_uuid`."```
just like 2 lines below 
```
The `audit` query parameter.
```